src/main/java/frc/robot/SwerveModule.java

```diff
@@ -152,23 +149,23 @@ public SwerveModulePosition getModulePos() {
      * @return Returns number rotations of translation motor BEFORE GEAR RATIO
      */
     public double getTransPositionRaw() {
-        return _transEncoder.getPosition();
+        return mTransEncoder.getPosition();
     }
 
     /**
      * 
      * @return Returns rotation position in radians
      */
     public double getRotPositionRaw() {
-        return _rotEncoder.getAbsolutePosition();
+        return mRotEncoder.getAbsolutePosition();
```

Comment:
A defect here? the function says return value is radians.
However mRotEncoder.getAbsolutePosition() return degrees by default. Where is the conversion from degrees to radians logic?

Reply:
See https://store.ctr-electronics.com/content/api/java/html/classcom_1_1ctre_1_1phoenix_1_1sensors_1_1_c_a_n_coder.html#a6f3d322d9755702e75da4273f63e587e

Reply:
When we configured the CANCoders, we set their conversion factors to radians. In the future, we need to figure out how to set that conversion in the code itself, but for now(and this PR) that value is in radians. We tested this code on Saturday and it all works, which is a testament to this fact.

Reply:
Change to use code to set conversion factor. we lost days in summer because we manually setup conversions inconsistently between 4 modules. It is easy to make inconsistent changes when we have to manually plug cable to each module and do by hand then to use the coding approach. If a CANCoders is broken at competitions, making a coding change now will gain back the time during competitions.

Reply:
We did not lose days in the summer because we set up conversions inconsistently, we lost days because the coding approach was making the conversions inconsistent, followed by the electrical/mechanical issues we had. That being said, I think we can add the coding approach in a new PR since that will take time to figure out and ensure it's working. I'll add an issue for this in Github.

Reply:
We did lose days in summer. One of those days is the Farmers Market Day. During that day electrical/mechanical were okay, at least for the drivebase, but the swerve module were not working. The bot was not drive-able. We already delayed during summer... why delaying again while the fix is just 5 lines of code: 

 ```suggestion
        CANCoderConfiguration steerEncoderConfig = new CANCoderConfiguration();
        steerEncoderConfig.absoluteSensorRange = AbsoluteSensorRange.Unsigned_0_to_360;
        steerEncoderConfig.initializationStrategy = SensorInitializationStrategy.BootToAbsolutePosition;
        steerEncoderConfig.magnetOffsetDegrees = Units.radiansToDegrees(steerAngleOffsetRad);
        mRotEncoder.configAllSettings(steerEncoderConfig);
```

and just one line in  getRotPosition()
```suggestion
        return 2*PI * mRotEncoder.getAbsolutePosition() / 360.0;
```

Reply:
The suggested code is for idea. One need to copy it to the right place where we initialize the steering encoder mRotEncoder.

Reply:
I already wrote this code when we first began switching to mk4 and we couldn't figure out what the problem The purpose of this PR is to bring the main up to date with what we accomplished with development, and I would argue this still requires more than 5 lines of code and sufficient testing, which justifies a separate PR. Could we finish merging in what we have before we add things?